WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -314,3 +315,46 @@ $venue-map-wrapper-height: 400px;
     background-color: inherit;
   }
 }
+
+table.table-condensed.admin-competitions {
```

Comment:
I think that I doesn't make sense to restrict the style to `.table-condensed`. If someone removes the class, the `.admin-competitions` no longer applies.